.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think we should be checking `.idea/` in

Reply:
+1

Reply:
+1 , please remove the `.idea/` folder

Reply:
Feel free to add `.idea` to `.gitignore` to preview this in the future